test/integration/services/data/test-get-app-court-reports.js

```diff
@@ -0,0 +1,37 @@
+const expect = require('chai').expect
+const knex = require('../../../../knex').appSchema
+
+const helper = require('../../../helpers/data/app-court-reports-helper')
+const getAppCourtReports = require('../../../../app/services/data/get-app-court-reports')
+
+var inserts = []
+
+describe('services/data/get-app-court-reports', function () {
+  before(function () {
+    return helper.insertDependencies(inserts)
+      .then(function (builtInserts) {
+        inserts = builtInserts
+      })
+  })
+
+  it('should retrieve all the court-reports with staging ids within a given range and the given workload report id', function () {
+    return knex('court_reports').max('staging_id AS maxId').first()
+    .then(function (maxId) {
+      return knex('workload_report').whereNull('effective_to').first('id')
```

Comment:
Use the `workload_report_id` from inserts rather than DB?

Reply:
I'm not inserting a workload_report_id.  Think in general we shouldn't because then we would have 2 open workload_reports

Reply:
True, this might become an issue with the indexing. But what if there isn't one in the DB?